httpd/handlers/loginSocial.go

```diff
@@ -18,7 +18,7 @@ func LoginSocial(w http.ResponseWriter, r *http.Request) {
 		"github": {
 			"clientID":     os.Getenv("OAUTH_ID"),
 			"clientSecret": os.Getenv("OAUTH_SECRET"),
-			"redirectURL":  "http://syd.jjcm.org:9000/login-social/callback",
+			"redirectURL":  "https://api.non.io/login-social/callback",
```

Comment:
Might be worth abstracting this into a config.json in the proj. Or yaml. Or whatevas.

Reply:
yeah, and OAUTH_ID and OAUTH_SECRET are specifically for github auth, so I feel like those will need to be named better once we unlock a few more providers.